### PR TITLE
Add VSCode default build task to run formatter via Makefile

### PR DESCRIPTION
**Pull request by cspotcode**
## What it does

Pressing Ctrl+Shift+B in VSCode will now format all files, including the stuff we wired up to sort imports and add `from __future__ import annotations`

This requires you to have `make` installed which can be done in about a minute by following the instructions in the README under "Install make"

https://github.com/CastIronChat/twisted-metal#development

This PR also applies the formatting changes to all files.

## How it works

VSCode lets you describe common development tasks in a `tasks.json` file.  I've added one that tells VSCode to execute `make fmt` when you press Ctrl+Shift+B.  `make` is an industry-standard tool for running common development tasks such as formatting and automated tests.  These tasks are already described in our `Makefile`.

### CODE REVIEW

**Comment by cspotcode** [comment]
[Wire up `make fmt` as a VSCode "Task" so that Ctrl+Shift+B will run it](https://trello.com/c/YaTuTZHW/115-wire-up-make-fmt-as-a-vscode-task-so-that-ctrlshiftb-will-run-it)

**Comment by cspotcode** [comment]
[Wire up `make fmt` as a VSCode "Task" so that Ctrl+Shift+B will run it](https://trello.com/c/YaTuTZHW/115-wire-up-make-fmt-as-a-vscode-task-so-that-ctrlshiftb-will-run-it)

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Format all code",
+            // Ctrl-Shift-B will run this,
+            // because it is the default build task
```
Comment (Thorakii) [inline .vscode/tasks.json:7]: Thanks for adding these comments

**Review by Thorakii — CHANGES_REQUESTED** [review]
I installed scoop and make following the instructions on the main page. Running `make fmt` in this branch throws this error for me "No module named isort make: *** [Makefile:9: fmt] Error 1"
I probably installed something wrong, or in the wrong place. If you can walk me through getting this working on my machine, and possibly post the resulting instructions so others of my knowledge level can do the same, this PR is ready for merge

**Review by Thorakii — DISMISSED** [review]
Thanks, Running `pip install -r requirements.txt` in the VS code terminal fixed the issue. Approved

**Review by Thorakii — APPROVED** [review]
Approved. Main was merged in.

**Pull request merged**